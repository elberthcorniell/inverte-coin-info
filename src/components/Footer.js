const Footer = () => <footer>
    <em>Developed by Elbert Corniell</em>
</footer>

export default Footer;
