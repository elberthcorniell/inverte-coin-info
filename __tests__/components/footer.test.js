import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '../../src/components/Footer';

describe('test', () => {
    test('Render as spected', () => {
        render(<Footer />);
        expect(document.body.innerHTML).toMatchSnapshot();
    });

});
