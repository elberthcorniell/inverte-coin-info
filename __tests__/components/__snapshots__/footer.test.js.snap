// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Render as spected 1`] = `"<div><footer><em>Developed by Elbert Corniell</em></footer></div>"`;
